Bump pytest from 7.2.0 to 8.0.1 (#15)
IBM/detect-secrets-stream: README.md

## Description

> Detect Secrets Stream is a server tool which ingests metadata of all (public repositories by default, private repositories are opt-in only) git pushes on your company's GitHub server. For each push, it [scans](./docs/detect-secrets-suite-faq.md#what-file-content-do-you-scan) the push contents for secrets. Once found and verified, secrets metadata will be stored in a database, and the raw secret will be stored in SOS Vault.

Reviewer: ```suggestion
Detect Secrets Stream is a server tool which ingests metadata of all (public repositories by default, private repositories are opt-in only) git pushes on your company's GitHub Enterprise server. For each push, it [scans](./docs/detect-secrets-suite-faq.md#what-file-content-do-you-scan) the push contents for secrets. Once found and verified, secrets metadata will be stored in a database, and the raw secret will be stored in Vault.
```

## Description

Detect secrets stream is the server tool which ingests metadata of all (public repo by default, private repo opt-in) git pushes which happen on your company's github server. For each push, it [scans](./docs/about-detect-secrets.md#what-file-content-do-you-scan) the push content for secrets. Once found and verified, secrets will be stored in db (meta data) and SoS vault (raw secret).

> The live secrets are reported to the Vulnerability Management team (VMT). Currently, the reporting is through csv file uploaded in shared box folder. We are about to make a change to report through a read-only DB table.

Reviewer 2: This line can be removed or edited to be non-IBM-specific.

Reviewer 2: Also the read-only DB table is implemented now.

Reviewer 3: https://github.com/IBM/detect-secrets-stream/pull/15/commits/6de0130a83c72755628551ed549d3a363ca49f86

### End 2 end test

This requires a personal or staging environment. See [kustomize_envs/dev/README.md](kustomize_envs/dev/README.md) for more details.

## Utilities

This repo has provided a utility module which enables an admin to do many routine tasks. The utility is invoked though `python -m detect_secrets_stream.util.secret_util`

> Running the utility requires several environment variables. You will first need to unlock secrets using `git-crypt`, then export the required environment variables. An example environment variable file (`.env.example`) has been prepared to help you.

Reviewer: ```suggestion
Running the utility requires several environment variables. An example environment variable file (`.env.example`) has been provided.
```

From a `fish` shell, you can do something like below

```shell

> cp .env.example .env.prod

Reviewer: There is a better way to expose env var with [`direnv`](https://direnv.net/)   That can be added in future PR.
